calico-enterprise/reference/component-resources/configure-resources.mdx

```diff
@@ -0,0 +1,1128 @@
+---
+description: Configure Resource requests and limits.
+---
+
+# Configure resource requests and limits
+
+## Big picture
+
+Resource requests and limits are essential configurations for managing resource allocation and ensuring optimal performance of Kubernetes workloads. In {{prodname}}, these configurations can be customized using custom resources to meet specific requirements and optimize resource utilization.
+
+:::note
+It's important to note that the CPU and memory values used in the examples are for demonstration purposes and should be adjusted based on individual system requirements.
+:::
+
+
+
+## APIServer
```

Comment:
```suggestion
## APIServer custom resource
```
Change globally for CRs. Helps visually to distinguish the major buckets from smaller types.